subsys/bluetooth/shell/flash.c

```diff
@@ -55,15 +61,15 @@ static int cmd_erase(int argc, char *argv[])
 	result = flash_erase(flash_dev, page_addr, size);
 
 	if (result) {
-		printk("Erase Failed, code %d.\n", result);
+		error(shell, "Erase Failed, code %d.\n", result);
 	} else {
-		printk("Erase success.\n");
+		print(shell, "Erase success.\n");
```

Comment:
I think `print` macro already adds `\r\n` at the end of a message.

So you will have one extra empty line.